tests/by-util/test_du.rs

```diff
@@ -1295,3 +1295,27 @@ fn test_du_inodes_blocksize_ineffective() {
             );
     }
 }
+
+#[test]
+fn test_du_inodes_total_text() {
+    let ts = TestScenario::new(util_name!());
+    let at = &ts.fixtures;
+
+    at.mkdir("d");
+    at.mkdir("d/d");
```

Comment:
You can simplify it to:
```suggestion
    at.mkdir_all("d/d");
```